### ISSUE TITLE
Lands of Lore 2

### ISSUE THREAD

**Issue by SuperSirLink** [schellingb/dosbox-pure#237]
I am not able to get this game to load.  I am getting an error about not enough space for SwapMin - 20971520 fatal error 1011 VMM initialization error 1.  I sourced this game from GOG and it launches when I use their shortcut.  Looking at the conf file it appears they are launching lolg.exe with a -cd parameter...

**Comment by SuperSirLink** [comment]
yeah I figured that.  I suspect it is trying to write to what would be the c drive and that it is larger than the file size limit for the same file redirect.  That is what I suspect the -cd switch is doing, but even putting that command in a batch file renders the same result.

**Comment by cyberconan** [comment]
Hi!
I had same issue and after several hours I have found a solution with the including of this line in .bat file:
set DOS4GVM=SwapMin:24M,SwapInc:0
Regards!
